src/tenant/tenant.service.ts

```
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
+import { CreateTenantDto } from './dto/create-tenant.dto';
+import { UpdateTenantDto } from './dto/update-tenant.dto';
+
+@Injectable()
+export class TenantService {
+  create(createTenantDto: CreateTenantDto) {
```

ESLint

[error] 'createTenantDto' is defined but never used.

(@typescript-eslint/no-unused-vars)